### DISCUSSION TITLE
Slack

### DISCUSSION THREAD

**Discussion by kubraaksux** [bounswe/bounswe2023group1#20] [Ideas]
Hey, also why dont we consider using slack? What you think?

**Comment by furknbulbul** [comment]
Free version of slack has only messages-files that are sent last 90 days. Given that we will be using the material next semester it might   be useless.

**Reply by volcaniqueo** [reply]
I agree on this comment.

**Comment by kubraaksux** [comment]
Oh that may be a problem but can be sufficient for this term too. I think its great vision for real company meetings. 
I checked last year's communication plans. I think we should have a communication plan and a description of this in our home page. Can i assign you @HarunErgen for this because you were the assignee of editing home page issue? But if we have any other volunteer, that will be also very fine!

**Reply by HarunErgen** [reply]
I can handle it.

**Reply by kubraaksux** [reply]
thanks a lot. may be better after deciding the communication plan or if you want you can create the draft 
#35 #36 #37 #38
